src/ext/java/org/opentripplanner/ext/emissions/DigitransitEmissionsService.java

```diff
@@ -0,0 +1,109 @@
+package org.opentripplanner.ext.emissions;
+
+import java.util.HashMap;
+import java.util.List;
+import org.opentripplanner.ext.flex.FlexibleTransitLeg;
+import org.opentripplanner.framework.lang.Sandbox;
+import org.opentripplanner.model.plan.Itinerary;
+import org.opentripplanner.model.plan.ScheduledTransitLeg;
+import org.opentripplanner.model.plan.StreetLeg;
+import org.opentripplanner.model.plan.TransitLeg;
+import org.opentripplanner.street.search.TraverseMode;
+
+@Sandbox
+public class DigitransitEmissionsService implements EmissionsService {
+
+  private HashMap<String, DigitransitEmissionsAgency> emissionByAgency;
```

Comment:
Then, for car, the key could just be `CAR` because the agency is not relevant. Although, it could also make sense to have a special field here to store the car emission as we don't at least yet need to support emissions for other street modes.